lib/mayaUsd/ufe/Global.cpp

```diff
@@ -112,6 +112,17 @@ MStatus initialize()
     Ufe::RunTimeMgr::instance().setContextOpsHandler(g_MayaRtid, proxyShapeContextOpsHandler);
 #endif
 
+#if UFE_PREVIEW_VERSION_NUM >= 2028
```

Comment:
Update to @williamkrick changes in UFE.

Reply:
This is already in the dev branch. We should have brought it with a merge to avoid conflicts.